lnurl/explainer.md

```diff
@@ -0,0 +1,178 @@
+# LNURL
+
+LNURL is a standard for streamlining the process of enabling a user to conduct certain Lightning Network based operations with a service or full node server on the network. It is particularly useful because it automates some otherwise manual steps when executing these operations between two or more parties. This allows providers that make use of Lightning Network functionality to deploy automated versions of their offerings that can interact autonomously with users by implementing one or more of the various flows.
```

Comment:
```suggestion
LNURL is a standard for streamlining the process of enabling a client/consumer to conduct certain Lightning based operations with a web service that is connected to the Lightning Network. The client can be a wallet application, or even another web service. LNURL Flows automate some otherwise manual steps when executing these operations between the two parties. This allows web service providers that make use of Lightning Network functionality to deploy API endpoints that can be interacted autonomously with by the clients which implementing one or more of the various flows.
```

Reply:
Yea I think we should clean this up too. I saw in the LNURL docs they use `LN SERVICE` and `LN WALLET`. I generally like this for the current state of things, and the only place I can see this potentially breaking down is if LNURL eventually supports wallet-to-wallet interactions.

To kick things off I can cleanup all the different references to the various actors as one of just these two options and then we can decide if these are the right nouns or if we'd like anything else. What do you think?